modules/polkadot-bridge/src/lib.rs

```diff
@@ -36,17 +36,28 @@ use support::{
 pub use module::*;
 
 /// The params related to rebalance per era
-#[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, Eq, Default)]
-pub struct SubAccountStatus {
+#[derive(Encode, Decode, Clone, RuntimeDebug, PartialEq, Eq)]
+pub struct SubAccountStatus<T: Config> {
 	/// Bonded amount
 	pub bonded: Balance,
 	/// Free amount
 	pub available: Balance,
 	/// Unbonding list
-	pub unbonding: Vec<(EraIndex, Balance)>,
+	pub unbonding: BoundedVec<(EraIndex, Balance), T::MaxUnbonding>,
 	pub mock_reward_rate: Rate,
 }
 
+impl<T: Config> Default for SubAccountStatus<T> {
```

Comment:
the derived Default doesn't work?

Reply:
No, it fails due to unsatisfied trait bounds

Reply:
Right. The issue is it will also add `T: Default` which doesn't make sense. The workaround is avoid pass `Config` directly but pass `MaxUnbonding` instead